scripts/deprecate-old-versions.sh

```diff
@@ -15,7 +15,7 @@ if [ -z "$OTP" ]; then
 fi
 
 with_replacement() {
-  npm deprecate $1 'The `'"$1"'` package is part of Apollo Server v2 and v3, which are now deprecated (end-of-life October 22nd 2023). This package'"'"'s functionality is now found in the `'"$2"'` package. See https://www.apollographql.com/docs/apollo-server/previous-versions/ for more details.' --otp=$OTP &
+  npm deprecate $1 'The `'"$1"'` package is part of Apollo Server v2 and v3, which are now deprecated (end-of-life October 22nd 2023 and October 22nd 2024, respectively). This package'"'"'s functionality is now found in the `'"$2"'` package. See https://www.apollographql.com/docs/apollo-server/previous-versions/ for more details.' --otp=$OTP &
```

Comment:
Suggested edits to be more readable/scannable: 

```
The `'"$1"'` package is used in both Apollo Server v2 and v3, which are now deprecated. Please note the End-of-life dates:

- Apollo Server v2: End-of-life on October 22nd, 2023
- Apollo Server v3: End-of-life on October 22nd, 2024

This package'"'"'s functionality is now found in the `'"$2"'` package. See https://www.apollographql.com/docs/apollo-server/previous-versions/ for more details.
```

Reply:
We decided offline to stick with what we have (just to avoid taking up extra log space with newlines / not sure if they're supported). We can always revisit and change this if we choose to.